xtable-core/src/test/java/org/apache/xtable/ITConversionController.java

```diff
@@ -103,6 +109,7 @@ public class ITConversionController {
 
   private static JavaSparkContext jsc;
   private static SparkSession sparkSession;
+  private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
use `UPPER_UNDERSCORE` when defining constants and move this up to line 109 to group with the `DATE_FORMAT`

Reply:
> use `UPPER_UNDERSCORE` when defining constants and move this up to line 109 to group with the `DATE_FORMAT`

Done! Thanks for the suggestion